1.0/docs/devguide/templates.md

```diff
@@ -303,6 +303,28 @@ If you or a third-party library mutate the array without Polymer's methods,
 you need to call [`notifySplices`](#notifysplices) to ensure that any elements
 watching the array are properly notified.
 
+### Improve performance for large lists {#large-list-perf}
+
+By default, `dom-repeat` tries to render all of the list items at once. If 
+you try to use `dom-repeat` to render a very large list of items, the UI may 
+freeze while it's rendering the list. If you encounter this problem, enable 
+"chunked" rendering by setting 
+[`initialCount`](/1.0/api/#dom-repeat:property-initialCount). In chunked mode, 
```

Comment:
These links aren't working correctly. Current workaround is to add a `target` attribute to the link (you can do that in markdown by adding `{:target="_blank"}` after the link, like the render link above (L288).

The problem here is that for normal page transitions, we usually try to inject the page contents into the current page rather than loading a whole new page ... But that doesn't work for the API docs, which are basically a separate app.

For consistency, "1.0" in the URL should probably be replaced with `{% polymer_version_dir %}` as in the previous link.